week-3/todo-list/style.css

```diff
@@ -1 +1,48 @@
+:root {
+  font-size: xx-large;
+}
 
+body {
+  background-image: url(assets/pattern_squares-2_1_4_0-0_180_2__ee6d4d_293241_e0fbfc_3d5a80_98c1d9.png);
+}
+
+.content {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+input {
+  font-size: xx-large;
+  border: solid 2px #293241;
+  border-radius: 1rem;
+  padding: 15px 32px;
+  margin-top: 1rem;
+}
+
+button {
+  background-color: #293241;
+  border-color: aliceblue;
+  font-size: large;
+  border-radius: 1rem;
+  color: white;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  margin-top: 1rem;
+}
+.done {
+  text-decoration: line-through;
+}
+
+.list-group {
+  background-color: #293241;
+  font-size: xx-large;
+  border-radius: 1rem;
+  border: solid;
+  color: white;
+  padding: 10px 25px;
+  text-align: center;
+}
```

Comment:
Some really nice styles, consistant theming etc, no issues I can see except that the delete and completed buttons might be a bit nicer with a bit of spacing